src/msg/CMakeLists.txt

```diff
@@ -57,6 +57,10 @@ if(WITH_DPDK)
     ${async_dpdk_srcs})
   target_link_libraries(common_async_dpdk PRIVATE
     dpdk::dpdk)
+  find_package(numa QUIET)
```

Comment:
can we assume that libnuma is linked explicitly only if libdpdk is built from the submodule? if that's the case, probably we can move the numa stuff to `BuildDPDK.cmake`? and add this linkage to `dpdk::eal` instead ?